### PR TITLE
Fix application create error

### PR DESCRIPTION
**Pull request by containscafeine**
This commit moves the logic of checking if an application exists
outside of SetCurrent()

### CODE REVIEW

**Comment by kadel** [comment]
👍 thanks for catching that

File: pkg/application/application.go [hunk 1]
```diff
@@ -201,3 +183,17 @@ func SetCurrent(name string) error {
 
 	return nil
 }
+
+func Exists(name string) error {
+	apps, err := List()
+	if err != nil {
+		return errors.Wrap(err, "unable to get active application")
```
Comment (kadel) [inline pkg/application/application.go]: error message is not correct :-(

File: pkg/application/application.go [hunk 2]
```diff
@@ -201,3 +183,17 @@ func SetCurrent(name string) error {
 
 	return nil
 }
+
+func Exists(name string) error {
```
Comment (kadel) [inline pkg/application/application.go]: shouldn't this also return bool?

It is not possible to determine if application doesn't exist or if there were error while checking it

File: pkg/application/application.go (hunk 1 above)
Comment (containscafeine) [reply pkg/application/application.go]: fixed

File: pkg/application/application.go (hunk 2 above)
Comment (containscafeine) [reply pkg/application/application.go]: correct, it should return both

Comment (containscafeine) [reply pkg/application/application.go]: fixed

**Pull request merged**